### PR TITLE
Add test for backend search

### PR DESCRIPTION
**Pull request by JaurbanRH**

### CODE REVIEW

**Comment by JaurbanRH** [comment]
Quality check failed because of missing init file which is in PR #352

**Comment by dhlavac** [comment]
@JaurbanRH why you don't include the init file in this PR as well?

File: testsuite/tests/ui/search/test_backend_search.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+"""
+Rewrite of spec/ui_specs/users/users_search_spec.rb
```
Comment (dhlavac) [inline testsuite/tests/ui/search/test_backend_search.py:2]: Imho leftover from user search test

**Comment by JaurbanRH** [comment]
#352 needs to be merged first.

**Review by jsmolar — APPROVED** [review]

**Review by dhlavac — CHANGES_REQUESTED** [review]
Same issue as PR #352

File: testsuite/tests/ui/search/test_backend_search.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+@pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-8562")
+def test_search_product(login, navigator, request, custom_ui_backend):
+    """
+    Preparation:
+        - Create custom account
+        - Create custom application
+    Test if:
+        - you search account by org_name it will return the correct one
+        - you search account by username it will return the correct one
+        - you search account by email it will return the correct one
```
Comment (dhlavac) [inline testsuite/tests/ui/search/test_backend_search.py:21]: Incorrect test description, backends are not searchable by email or application

File: testsuite/tests/ui/search/test_backend_search.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+import pytest
+
+from testsuite.ui.views.admin.backend import BackendsView
+from testsuite.utils import blame
+
+
+# pylint: disable=unused-argument
+@pytest.mark.xfail
+@pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-8562")
+def test_search_product(login, navigator, request, custom_ui_backend):
```
Comment (dhlavac) [inline testsuite/tests/ui/search/test_backend_search.py:13]: Incorrect name of the test

File: testsuite/tests/ui/search/test_backend_search.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        assert backends.table.row()[1].text == sys_name
+
+
+# pylint: disable=unused-argument
+def test_search_multiple_backends(login, navigator, custom_backend, request):
+    """
+    Preparation:
+        - Create 4 custom backends
+    Test if:
+        - you search account by base name it will return the correct ones (first 3)
```
Comment (dhlavac) [inline testsuite/tests/ui/search/test_backend_search.py:41]: search of backend not account

File: testsuite/ui/views/admin/backend/__init__.py [hunk 5]
```diff
@@ -14,6 +14,13 @@ class BackendsView(BaseAdminView):
     table = PatternflyTable("//*[@id='backend-apis']/section/table", column_widgets={
         "Name": Text("./a")
     })
+    search_bar = TextInput(locator="//input[@type='search']")
+    search_button = GenericLocatorWidget("//button[contains(@aria-label,'search')]")
+
+    def search(self, value: str):
+        """Search in account table by given value"""
```
Comment (dhlavac) [inline testsuite/ui/views/admin/backend/__init__.py:21]: Search in  backend table

File: testsuite/tests/ui/search/test_backend_search.py [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    backends.search(f"{name}_name2")
+    assert backends.table.row()[0].text == f"{name}_name2"
+
+
+def test_search_non_existing_value(request, login, navigator, custom_backend):
+    """
+    Preparation:
+        - Create custom backend
+    Test if:
+        - you search account by non-existing-value it won't return anything
```
Comment (dhlavac) [inline testsuite/tests/ui/search/test_backend_search.py:63]: Backend no account

**Review by dhlavac — CHANGES_REQUESTED** [review]
PR still contains changes from #352 please remove them from this PR or apply coments from #352 to this PR as well

**Pull request closed without merging**